### PR TITLE
Rename OpenJ9 MethodType fields to match OpenJDK MethodType

### PR DESCRIPTION
**Pull request by babsingh**
1. "arguments" renamed to "ptypes".
2. "returnType" renamed to "rtype".

Related: #7352

Co-authored-by: Jack Lu <Jack.S.Lu@ibm.com>
Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

**Comment by babsingh** [comment]
fyi @fengxue-IS @DanHeidinga

**Comment by DanHeidinga** [comment]
Jenkins test sanity xlinux jdk8,jdk11,jdk15

**Comment by DanHeidinga** [comment]
@0xdaryl can you take a look at the jit changes?  I think they're OK but would appreciate confirmation that there isn't some assumption baked in on the name length, or ...

File: runtime/compiler/il/J9Symbol.cpp [hunk 1]
```diff
@@ -142,7 +142,7 @@
        {r(TR::Symbol::Java_lang_invoke_MutableCallSiteDynamicInvokerHandle_mutableSite,"java/lang/invoke/MutableCallSiteDynamicInvokerHandle", "mutableSite", "Ljava/lang/invoke/MutableCallSite;")},
        {r(TR::Symbol::Java_lang_invoke_MethodHandle_thunks,           "java/lang/invoke/MethodHandle", "thunks", "Ljava/lang/invoke/ThunkTuple;")},
        {r(TR::Symbol::Java_lang_invoke_MethodHandle_type,             "java/lang/invoke/MethodHandle", "type", "Ljava/lang/invoke/MethodType;")},
-       {r(TR::Symbol::Java_lang_invoke_MethodType_arguments,          "java/lang/invoke/MethodType", "arguments", "[Ljava/lang/Class;")},
+       {r(TR::Symbol::Java_lang_invoke_MethodType_ptypes,          "java/lang/invoke/MethodType", "ptypes", "[Ljava/lang/Class;")},
```
Comment (0xdaryl) [inline runtime/compiler/il/J9Symbol.cpp:145]: Could you keep the string indentation consistent with the surrounding code please?

**Review by 0xdaryl — CHANGES_REQUESTED** [review]
Looks good.  Only one minor formatting request.

**Comment by DanHeidinga** [comment]
The build failures look to be jenkins related.  @babsingh once you fix the formatting as requested by Daryl, I'll relaunch the builds

File: runtime/compiler/il/J9Symbol.cpp (hunk 1 above)
Comment (babsingh) [reply runtime/compiler/il/J9Symbol.cpp]: Yes, the indentation has been fixed.

**Comment by babsingh** [comment]
Tested a local build with OpenJDK MHs enabled; these changes work fine; WIP removed.

**Comment by DanHeidinga** [comment]
Jenkins test sanity xlinux jdk8,jdk11,jdk15

**Comment by DanHeidinga** [comment]
@0xdaryl are you good with the updated formatting?

**Review by 0xdaryl — APPROVED** [review]

**Review by DanHeidinga — APPROVED** [review]

**Pull request merged**